docs/architecture/grpc-for-wcf-developers/client-libraries.md

```diff
@@ -69,7 +69,7 @@ public class GrpcStreamSubscription : IDisposable
     private readonly CancellationTokenSource _tokenSource;
     private bool _completed;
 
-    public Subscription(IAsyncStreamReader<T> reader, IObserver<T> observer, CancellationToken token)
+    public GrpcStreamSubscription(IAsyncStreamReader<T> reader, IObserver<T> observer, CancellationToken token)
     {
         Debug.Assert(reader != null);
         Debug.Assert(observer != null);
```

Comment:
I think these should be throw instead of assertions. I'll wait for confirmation before changing this.

I also can't get the value that `_tokenSource` defined in the next line adds? I think that field is unnecessary.